lib/internal/modules/esm/hooks.js

```diff
@@ -329,10 +382,13 @@ class Hooks {
    * until it reaches the bottom or short-circuits.
    * @param {URL['href']} url The URL/path of the module to be loaded
    * @param {object} context Metadata about the module
+   * @param {number} [chainId]
+   * @param {number} [threadId]
+   * @param {any} [workerData]
    * @returns {Promise<{ format: ModuleFormat, source: ModuleSource }>}
    */
-  async load(url, context = {}) {
-    const chain = this.#chains.load;
+  async load(url, context = {}, chainId = undefined, threadId = undefined, workerData = undefined) {
```

Comment:
Is it necessary to assign a default value of `undefined` here?

Reply:
Actually not. I'll fix it.